processing/src/main/java/org/apache/druid/query/groupby/GroupByQueryConfig.java

```diff
@@ -368,6 +392,8 @@ public GroupByQueryConfig withOverrides(final GroupByQuery query)
         CTX_KEY_ENABLE_MULTI_VALUE_UNNESTING,
         isMultiValueUnnestingEnabled()
     );
+
+    logger.debug(newConfig.toString());
```

Comment:
nit: remove or add something in the log message that explains what the debug message is

```suggestion
```